src/core/sys/posix/sys/stat.d

```diff
@@ -1078,36 +1078,82 @@ else version (Darwin)
 }
 else version (FreeBSD)
 {
+    import core.sys.freebsd.config;
+
     // https://github.com/freebsd/freebsd/blob/master/sys/sys/stat.h
+    static if (__FreeBSD_version >= INO64_FIRST)
+    {
+        struct stat_t
+        {
+            dev_t     st_dev;
+            ino_t     st_ino;
+            nlink_t   st_nlink;
+            mode_t    st_mode;
+            short st_padding0;
+            uid_t     st_uid;
+            gid_t     st_gid;
+            int st_padding1;
+            dev_t     st_rdev;
+
+            version (X86) int st_atim_ext;
 
-    struct stat_t
+            time_t      st_atime;
+            c_long      __st_atimensec;
```

Comment:
It seems to me that these really should just all be `timespec`s, since that's what they are in the C header file, but this is indeed what we have here already for some reason. So, I guess that that can be sorted out later.